src/platform/testing/e2e/nightwatch-commands/allyDeepEquals.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable func-names, prefer-arrow-callback */
+
+const focusable = import('ally.js/query/focusable');
+const tabbable = import('ally.js/query/tabbable');
+
+/**
+ * Checks if the objects of focusable and tabbable elements are equal.
+ * This is useful for pages that might include tabindex="-1" on one or
+ * more elements. In that case, we'd expect the two objects not to be
+ * equal. All other cases assume objects should be equal.
+ * https://allyjs.io/api/query/focusable.html
+ *
+ * "Tabbable" means the element or elements can receive
+ * keyboard focus natively by pressing `TAB` or `SHIFT + TAB`,
+ * and elements that have a tabindex="-1" attribute, and can be
+ * focused programmatically.
+ *
+ * @method allyDeepEquals
+ * @param {string} [selector] The selector (CSS / Xpath) used to locate the element.
+ * @returns {array} [comparator] The array objects for focusable, and tabbable elements.
+ * @api commands
+ */
+exports.command = function allyDeepEquals(selector, callback) {
```

Comment:
Is this command useful outside the context of the assertion? If not, consider moving inside the assertion function.